cypress/page/payment-step.page.ts

```diff
@@ -4,9 +4,9 @@ class PaymentStepPage {
   private orderComplete: string;
 
   constructor() {
-    this.payByBankBtn = "a.bankwire";
-    this.checkoutBtn = "button.button.btn.btn-default.button-medium";
-    this.orderComplete = "#center_column > div > p > strong";
+    this.payByBankBtn = 'a.bankwire';
+    this.checkoutBtn = 'button.button.btn.btn-default.button-medium';
```

Comment:
maybe this can work better: `#cart_navigation button[type=submit]`

Reply:
resolved